slo_generator/constants.py

```diff
@@ -18,25 +18,25 @@
 import os
 
 # Compute
-NO_DATA = -1
-MIN_VALID_EVENTS = int(os.environ.get("MIN_VALID_EVENTS", "1"))
+NO_DATA: int = -1
```

Comment:
I only annotated `NO_DATA` at first, then found the file was small enough that we can annotate everything in it in this PR.